samples/utils/CMakeLists.txt

```diff
@@ -40,4 +40,5 @@ target_link_libraries(utils PUBLIC glfw)
 target_link_libraries(utils PUBLIC glslang)
 target_link_libraries(utils PUBLIC glslang-default-resource-limits)
 target_link_libraries(utils PUBLIC SPIRV)
-target_compile_definitions(utils PUBLIC VULKAN_HPP_DISPATCH_LOADER_DYNAMIC=1)
\ No newline at end of file
+target_compile_definitions(utils PUBLIC VULKAN_HPP_DISPATCH_LOADER_DYNAMIC=1)
+target_compile_features(utils PUBLIC cxx_std_14)
```

Comment:
If c++14 is required for the samples travis won't find potential c++11 features.